host/xtest/xtest_6000.c

```diff
@@ -1459,6 +1463,126 @@ static void xtest_tee_test_6015_single(ADBG_Case_t *c, uint32_t storage_id)
 
 DEFINE_TEST_MULTIPLE_STORAGE_IDS(xtest_tee_test_6015)
 
+
+struct test_6016_thread_arg {
+	ADBG_Case_t *case_t;
+	uint32_t storage_id;
+	uint8_t file_name[8];
+};
+
+static void *test_6016_thread(void *arg)
+{
+	struct test_6016_thread_arg *a = arg;
+	TEEC_Session sess;
+	uint32_t obj;
+	uint8_t out[10] = { 0 };
+	uint32_t count;
+	uint32_t orig;
+
+	if (!ADBG_EXPECT_TEEC_SUCCESS(a->case_t,
+		xtest_teec_open_session(&sess, &storage_ta_uuid, NULL, &orig)))
```

Comment:
The concurrency would be improved if the sessions were created before the threads.